docs/cn/Key-Value-System-API.md

```diff
@@ -23,8 +23,8 @@ Alluxio除了提供[Filesystem API](File-System-API.html) 让应用程序来读
 
 ## 键值存储库配置参数
 
-Alluxio默认配置是禁用键值存储库的，可以通过配置`alluxio.keyvalue.enabled`为true来启用 (see
-[configuration parameters](Configuration-Settings.html))
+Alluxio 默认配置是禁用键值存储库的，可以通过配置`alluxio.keyvalue.enabled`为true来启用 （参考
```

Comment:
Could you remove the extra space here?
and use `git add -u`, `git commit -m "remove space"` and `git push origin smallfix` to submit the changes?